core/market/decentralized/src/matcher/store.rs

```diff
@@ -58,14 +58,21 @@ impl SubscriptionStore {
             })
     }
 
-    pub async fn get_offers(&self, id: Option<Identity>) -> Result<Vec<ClientOffer>, OfferError> {
+    pub async fn get_offers(
+        &self,
+        id: Option<Identity>,
+        validation_ts: NaiveDateTime,
+    ) -> Result<Vec<ClientOffer>, OfferError> {
         Ok(self
             .db
             .as_dao::<OfferDao>()
-            .get_offers(match id {
-                Some(ident) => Some(ident.identity),
-                _ => None,
-            })
+            .get_offers(
+                match id {
```

Comment:
consider also applying this hint from @nieznanysprawiciel 
https://github.com/golemfactory/yagna/pull/333#discussion_r451510883